Utils/github_workflow_scripts/slack_notifier_master/slack_notifier.py

```diff
@@ -50,11 +100,34 @@ def construct_slack_section(_section_title: str, _failed_entities: Collection[st
     if failed_jobs:
         slack_body_message.append(
             construct_slack_section(
-                _section_title="Failed Github-Actions Jobs",
+                "Failed Github-Actions Jobs",
                 _failed_entities=failed_jobs,
             )
         )
 
+    failed_unit_tests, failed_integration_tests, failed_graph_tests = failed_tests
+
+    if failed_unit_tests:
```

Comment:
does it make sense to do ... once it gets too long or too many failed?

Reply:
yes, slack truncates it automatically